### DISCUSSION TITLE
Search style between two dates

### DISCUSSION THREAD

**Discussion by bepolymathe** [freeplane/freeplane#936] [Help] [answered]
Hi all, 

I created a new map with nodes in a date format `12/01/2023` (which are well recognized as the date because I have the indicator "calendar" in the status bar). I would like to look for all the nodes between two dates. I thought I could do it with the filters but I can't. 

![image](https://user-images.githubusercontent.com/29939805/212171412-e5beac6a-ca49-4ec3-8347-73b41557fa4e.png)

Where did I make a mistake ?

PS: Same with node text instead all text

**Comment by bepolymathe** [comment]
It's weird I'm sure I've already managed to make such a filter and I can't do it anymore 🤔

**Comment by macmarrum** [comment]
You are asking the filter engine to look at text (Core, details or note), while what you apparently want is to look at dates (Date filter)

![Date_filter](https://user-images.githubusercontent.com/7197131/212415323-0acec940-0bd0-4315-b391-af0cbc51b635.png)

**Reply by bepolymathe** [reply]
Thanks for reply. My nodes look like this 

![image](https://user-images.githubusercontent.com/29939805/212416406-1e8e6bf9-377b-485d-97d0-836d3e75ccfc.png)

Date filter just allow to search "created" date and "modified" date.

**Reply by macmarrum** [reply]
You are right. I made a mistake.

I tried the date comparison and it doesn't work. Here's my bug report
- https://github.com/freeplane/freeplane/issues/945

**Comment by euu2021** [comment]
> ![image](https://user-images.githubusercontent.com/29939805/212171412-e5beac6a-ca49-4ec3-8347-73b41557fa4e.png)


Isn't this filter asking for dates that are BEFORE 09/01 AND AFTER 13/01?

**Reply by bepolymathe** [reply]
> > ![image](https://user-images.githubusercontent.com/29939805/212171412-e5beac6a-ca49-4ec3-8347-73b41557fa4e.png)
> 
> Isn't this filter asking for dates that are BEFORE 09/01 AND AFTER 13/01?

No, if had made a new test for my today’s notes... same result

![image](https://user-images.githubusercontent.com/29939805/212563422-750b7e99-b5b7-459d-b5c5-fcb5df761e17.png)

**Comment by ldmpub** [comment, marked as answer]
Should be fixed with Freeplane Preview 1.11.1_07 ; see https://github.com/freeplane/freeplane/issues/945.

**Reply by bepolymathe** [reply]
> Should be fixed with Freeplane Preview 1.11.1_07 ; see #945.

Yes thank you I saw that. I am waiting for version 1.11.1_07 to be available for testing... and confirm.

**Reply by dpolivaev** [reply]
https://sourceforge.net/projects/freeplane/files/freeplane%20preview/

**Reply by bepolymathe** [reply]
Oh... you're right i haven't looked in the right directory... 😆 
 
I confirm that it works very well. Thank you for everything !
